test/governance/Governor.test.js

```diff
@@ -40,7 +36,7 @@ contract('Governor', function (accounts) {
   for (const { mode, Token } of TOKENS) {
     describe(`using ${Token._json.contractName}`, function () {
       beforeEach(async function () {
-        this.chainId = await web3.eth.getChainId();
+        // this.chainId = await web3.eth.getChainId();
```

Comment:
Maybe we can assign the chainId the same value as in GovernorWithParams `this.chainId = web3.utils.toBN(246);` if that's not a failing reason, instead of commenting it.